pallav/fashionTech/pallavBanka/parkingAssignment/Parking.java

```diff
@@ -1,2 +1,173 @@
-package com.fashionTech.pallavBanka.parkingAssignment;
+package fashionTech.pallavBanka.parkingAssignment;
 
+import java.util.Date;
+import java.util.Timer;
+
+abstract class parking{
+    void parkVehicle(vehicle v){
+        int regno;
+        short floor;
+        int slotid;
+        regno=v.regno;
+        floor=v.floor;
+        slotid=v.slotid;
+    }
+    void getVehicle(vehicle v,int regno){
+        v.regno=regno;
+        if(v.outTime != null) {
+            //difference in time in milliseconds
+            long diff = v.outTime.getTime() - v.inTime.getTime();
```

Comment:
inTime/ OutTime should be mainted with the parking class, these are not properties of a vehicle.
Think of objects as real world objects.
When you park in parking lot, your bike won't store the time when it was parked, when it is getting out, the parking lot will maintain that information, same treatment you should give to Java object
@Girrajgg @PARIMALPRATYUSH keep above point in mind